docs/index.md

```diff
@@ -3,20 +3,19 @@
 <p align="center">
  <img src="https://raw.githubusercontent.com/andrewthederp/Disgames/main/docs/src/disgames.png" height="125px" width="125px" />
 </p>
-Welcome to the offical documentation of `disgames`! You can find our source code [here](https://github.com/andrewthederp/disgames).
+Welcome to the offical documentation of disgames! You can find our source code [here]((https://github.com/andrewthederp/disgames)).
 
 If you have any questions, check out our [faq](./faq.md)
 
 - [Disgames](#disgames)
-  - [Contents](#contents)
   - [Installation](#installation)
   - [Extra](#extra)
 
 ## Installation
 
 To install it from [pypi](https://pypi.org/project/disgames), run
 
-```sh
+```python
```

Comment:
this should be sh tho, its a shell command

Reply:
alr will fix but no difference